verticapy/_config/config.py

```diff
@@ -148,6 +153,205 @@ def set_option(key: str, value: Any = None) -> None:
         using iterative functions.
     value: object, optional
         New value of the option.
+
+    Examples
+    --------
+    Import and load the titanic dataset:
+
+    .. hint:: VerticaPy provides multiple datasets, all of which have loaders in the datasets module.
+
+    .. code-block:: python
+
+        from verticapy.datasets import load_titanic
+
+        titanic = load_titanic()
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        from verticapy.datasets import load_titanic
+        from verticapy import set_option
+
+        titanic = load_titanic()
+        html_file = open("figures/_config_config_set_option_1.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_1.html
+
+    Import the set_option function:
+
+    .. code-block:: python
+
+        from verticapy import set_option
+
+    Turn on the "count_on" option, which displays the total number of elements in the dataset:
+
+    .. code-block:: python
+
+        set_option("count_on", True)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("count_on", True)
+        html_file = open("figures/_config_config_set_option_2.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_2.html
+
```

Comment:
.. warning:: Exercise caution when enabling this option, as it may result in decreased performance. VerticaPy will perform calculations to determine the number of elements in a displayed vDataFrame, which can have an impact on overall system performance.